cycle.js

```diff
@@ -0,0 +1,156 @@
+for (let n = 2; n <= 10; n++) {
+    if (!(n % 2)) {
+        console.log(n);
+    }
+}
+
+let n = 0;
+while (n < 3) {
+    alert(`number ${n}!`);
+    n++;
+}
+
+let n;
+do {
+    n = prompt("Введите число больше 100", 0);
+}
+while (n <= 100 && n) {
+    alert(n);
+}
+
+let n = 10;
+outer:
+    for (let a = 2; a <= n; a++) {
+        for (let b = 2; b < a; b++) {
+            if (!(a % b)) continue outer;
+        }
+        alert(a);
+    }
+
+let n = 1;
+while (n <= 50) {
+    alert(n);
+    n++
+}
+
+for (let n = 1; n <= 50; n++) {
+    alert(n);
+}
+
+let n = 1;
+while (n <= 100) {
+    alert(n);
+    n++
+}
+
+for (let n = 1; n <= 100; n++) {
+    alert(n);
+}
+
+let n = 11;
+while (n <= 33) {
+    alert(n);
+    n++
+}
+
+for (let n = 11; n <= 33; n++) {
+    alert(n);
+}
+
+
+for (let n = 0; n <= 100; n++) {
+    if (n % 2 == 0) {
```

Comment:
remove == 0

you know how to write it in a proper way

Reply:
fixed